.github/workflows/ci.yml

```diff
@@ -245,60 +245,24 @@ jobs:
       - name: Run the tests
         run: invoke ci-tests '${{ matrix.markers }}' --up-services --verbose
 
-  py38-min-versions:
-    needs: [unit-tests, static-analysis]
-    if: github.event.pull_request.draft == false
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: "3.8"
-          cache: "pip"
-          cache-dependency-path: |
-            reqs/requirements-dev-test.txt
-      - name: Install dependencies
-        run: pip install . -c ci/constraints-test/py38-min-install.txt -r reqs/requirements-dev-test.txt
-      - name: Run the tests
-        run: invoke tests --unit
-
-  py39-min-versions:
-    needs: [unit-tests, static-analysis]
-    if: github.event.pull_request.draft == false
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: "3.9"
-          cache: "pip"
-          cache-dependency-path: |
-            reqs/requirements-dev-test.txt
-      - name: Install dependencies
-        run: pip install . -c ci/constraints-test/py39-min-install.txt -r reqs/requirements-dev-test.txt
-      - name: Run the tests
-        run: invoke tests --unit
-
-  py310-min-versions:
+  min-versions:
```

Comment:
I like the consolidation of this into 1 test stage.

I notice for this we use `invoke test` but for all stages we use `invoke ci-test`. Can you update that here?

Reply:
Done